woocommerce/woocommerce
# Add build-watch command for woocommerce-admin

### All Submissions:






### Changes proposed in this Pull Request:

I removed the `watch` part of all the js packages from the `build-watch woocommerce-admin` command. So that it stays more specific to the `woocommerce-admin` package.
To keep inline with this I added the `build-watch` to most of our js packages that `woocommerce-admin` merged over.

See this P2 thread for more context: pbIJXs-1oH-p2#comment-3602

### How to test the changes in this Pull Request:

1. Run `pnpm nx build-watch woocommerce-admin` it should run successfully and watch for changes in our client code
2. Make a change in the `client` code and see if it correctly is reflected
3. Run `pnpm nx build-watch @woocommerce/components`, it should build and watch correctly
4. Run `pnpm nx run-many --target=build-watch --projects=woocommerce-admin,@woocommerce/components` and see if it watches both places and updates correctly.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### FOR PR REVIEWER ONLY:
